### ISSUE TITLE
Whispers dont get colored when using /me

### ISSUE THREAD

**Issue by jeovanny315** [Chatterino/chatterino2#3253]
On the twitch website, whispers get colored but not on chatterino.

![image](https://user-images.githubusercontent.com/79304571/134613841-a30b2a65-72d1-47f0-9fb1-22fe177fb509.png)

![image](https://user-images.githubusercontent.com/79304571/134613853-78a5f8d3-b349-41a8-bc84-8ad42e9fb417.png)

Windows 11 Chatterino Nightly 7.3.4

**Comment by Mm2PL** [comment]
Whispers have /me ?!

**Comment by zneix** [comment]
I'm kinda surprised / relieved we got this sort of issue first before actually getting complaints about keeping coloured /me messages.

Jokes aside though, webchat has removed coloured action messages some time ago, @jeovanny315 are - you using some browser extension that could possibly try to emulate the behavior of action messages in whispers? I don't use webchat myself, but I'm pretty sure that vanilla webchat should give you the _cursive messages_ if anything. I'll test this bizzare case later though, thanks for bringing this up!

**Comment by jeovanny315** [comment]
Yeah, its ffz that colors /me on web. I totally forgot twitch removed that since ffz added it back like a day later.

**Comment by Mm2PL** [comment]
/me seems to still work in whispers even in clean webchat
